src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -4935,7 +4938,7 @@ void AddSC_generic_spell_scripts()
     new spell_gen_disabled_above_63();
     new spell_gen_black_magic_enchant();
     new spell_gen_area_aura_select_players();
-    new spell_gen_mount("spell_big_blizzard_bear", 0, SPELL_BIG_BLIZZARD_BEAR_60, SPELL_BIG_BLIZZARD_BEAR_100, 0, 0);
+    new spell_gen_mount("spell_big_blizzard_bear", 0, SPELL_BIG_BLIZZARD_BEAR_60, SPELL_BIG_BLIZZARD_BEAR_100, SPELL_BIG_BLIZZARD_BEAR_150, SPELL_BIG_BLIZZARD_BEAR_280, SPELL_BIG_BLIZZARD_BEAR_310);
```

Comment:
This should work and remove the extra `0, 0` at the end.
`new spell_gen_mount("spell_big_blizzard_bear", 0, SPELL_BIG_BLIZZARD_BEAR_60, SPELL_BIG_BLIZZARD_BEAR_100);`

Reply:
> This should work and remove the extra `0, 0` at the end.
> `new spell_gen_mount("spell_big_blizzard_bear", 0, SPELL_BIG_BLIZZARD_BEAR_60, SPELL_BIG_BLIZZARD_BEAR_100);`

Nope, doesn't work. Already tested like that, before opening this PR. 
1. Learning Apprentice riding -> Big Blizzard Bear **_(Inc speed by 60) - OK_**
2. Learning Journeyman riding ->  Big Blizzard Bear **_(Inc speed by 100) - OK_**
3. **_Learning Expert riding -> Issue (It revert it to slow speed)_**
4. **_Learning Artisan riding -> Same issue_**